template/chat/damage.html

```diff
@@ -10,7 +10,7 @@ <h3>{{name}}</h3>
     </div>
 
     <div class="wrapper-input">
-        <p><strong>{{localize "WEAPON.DAMAGE"}}: {{result.damage.total}}</strong></p>
+        <p><strong>{{localize "WEAPON.DAMAGE"}}: {{result.damage.total}}</span></strong></p>
```

Comment:
I don't think this `</span>` has a opening element

Reply:
Woops sorry, that was an artifact from older attempts. removed it now